.github/file-filters.yml

```
@@ -1,44 +1,69 @@
 # This is used by the action https://github.com/dorny/paths-filter
 
-# TODO: There are some meta files that we potentially could ignore for both front/backend,
-# as well as some configuration files that should trigger both
-frontend_components_lintable: &frontend_components_lintable
-  - '**/*.[tj]{s,sx}'
-
-frontend_lintable: &frontend_lintable
-  - *frontend_components_lintable
-  - '**/tsconfig*.json'
-  - '{package,now,vercel}.json'
-
-yarn_lockfile: &yarn_lockfile
-  - 'yarn.lock'
-
-eslint_config: &eslint_config
-  - '.eslint*'
-
-# `frontend_src` filters on files that are frontend changes excluding
-# changes to the tests/ directory.
-# If you want to filter on *all* frontend files, use the `frontend_all` filter.
+sentry_specific_workflow: &sentry_specific_workflow
+  - added|modified: '.github/workflows/frontend.yml'
+sentry_specific_test_files: &sentry_specific_test_files
+  - added|modified: 'tests/js/**/*'
+  - added|deleted|modified: 'fixtures/search-syntax/**/*'
+  - added|deleted|modified: 'fixtures/js-stubs/**/*'
+
+# Provides list of changed app files to lint against (stylelint)
+lintable_css_in_js_modified: &lintable_css_in_js_modified
+  - added|modified: 'static/**/*.[tj]{s,sx}'
+
+# Trigger for when we must run full lint (stylelint)
+lintable_css_in_js_rules_changed:
```

Should these be something like 

```suggestion
lintable_css_in_js_modified_or_rules_changed:
```

to be a bit more explicit